roxie/ccd/ccdkey.cpp

```diff
@@ -378,8 +378,10 @@ class InMemoryIndex : public CInterface, implements IInterface, implements IComp
             size32_t size = recordSize->getFixedSize();
             if (length % size)
```

Comment:
Not sure I see why it needs to - the limit is on there being more than 2^32 rows, not on the size being more than 2^32 bytes